clients/drcachesim/CMakeLists.txt

```diff
@@ -721,17 +721,20 @@ if (BUILD_TESTS)
            ${PROJECT_SOURCE_DIR}/clients/drcachesim/tests)
```

Comment:
Re the PR description:

Add a full description even if it repeats the title somewhat.

Reply:
Got it. Done.